designsafe/static/scripts/ng-designsafe/components/modals/data-browser-service-move/data-browser-service-move.component.js

```diff
@@ -1,34 +1,53 @@
 import DataBrowserServiceMoveTemplate from './data-browser-service-move.component.html';
 import _ from 'underscore';
+import { DataBrowserService } from '../../../services/data-browser-service';
+// import angular from 'angular';
 
 
 class DataBrowserServiceMoveCtrl {
 
-    constructor($scope, $state, FileListing, ProjectService) {
+    constructor($scope, $state, FileListing, ProjectService, DataBrowserService) {
         'ngInject';
         this.$scope = $scope
         this.$state = $state
         this.FileListing = FileListing
         this.ProjectService = ProjectService
+        this.DataBrowserService = DataBrowserService
     }
 
     $onInit() {
         this.files = this.resolve.files
         this.initialDestination = this.resolve.initialDestination
-    
+        console.log(this.DataBrowserService.currentState)
         this.data = {
             files: this.files
         };
         //$scope.data = data;
 
         this.listing = this.initialDestination;
+        this.position = this.$scope.currentOption = null;
+        let dbState = this.DataBrowserService.currentState;
+        if (dbState.listing.system == 'designsafe.storage.default') {
+          this.$scope.currentOption = this.options[2];
+        }
 
         this.state = {
             busy: false,
             error: null,
             listingProjects: false
         };
 
+        this.$scope.currentOption = null;
+        let dbState = this.DataBrowserService.currentState;
+        if (dbState.listing.system == 'designsafe.storage.default') {
+          this.$scope.currentOption = this.options[2];
+        };
```

Comment:
don't need this semicolon
```suggestion
        }
```